### ISSUE TITLE
Error when saving settings

### ISSUE THREAD

**Issue by Ntacman** [Enubia/ghost-chat#29]
**Checklist before reporting**
- [x] Is it really a bug or actually a feature request?
- Is there already another issue related to your problem?
- [x] Do you have resources to reproduce it?

**Describe the bug**
Application errors out when clicking the save button in the Settings area, but it seems settings still saved.

When
Any time you click the save button in the settings area, regardless of whether a change is made or not.

**To Reproduce**
1. Go to Settings in the right click menu
2. Click on the save button

**Expected behavior**
Settings should be saved without error

**Screenshots**
![CrashTrace](https://user-images.githubusercontent.com/26342364/84603439-16ba1080-ae54-11ea-8a45-aca4a1a289f0.PNG)

**Additional context**
I am using the latest release build, 1.1.2

**Comment by Ntacman** [comment]
Sorry for all the edits/additional comments, but I tested the standalone EXE after experiencing the above issues, and it does not seem to occur there. I then went back and tried the installed EXE, and cannot reproduce this anymore. Not sure if there was a config file or something created by the standalone that wasn't made by the installed version's executable?

**Comment by Enubia** [comment]
Indeed there's a config file created on initial startup, I think it just can't find the file though on the installer version. It's located in %appData%\Roaming\ghost-chat.

**Comment by Enubia** [comment]
Unfortunately I can't reproduce this error. I've added error logging in the upcoming release and fixed some config settings. So if this error occurs again, could you add the error on the issue? It will be located in %appdata%\Roaming\ghost-chat\logs

**Comment by Enubia** [comment]
I will keep this issue open for now, so just attach the log files on here :)

**Comment by Ntacman** [comment]
I'll give it a test in a clean VM to see if I can re-create it on a fresh install with the new version. Thanks for the quick response 👍

**Comment by Ntacman** [comment]
Tested 1.1.3 in a new windows 10 VM, and I didn't encounter this error again.

Here's what I tested

- [x] Installing for just my user
- [x] Installing for every user
- [x] Just for testing sake, I deleted the `config.json` file and saved settings afterwards. Ghost chat recreated the file without issue, no error.

I'm pretty sure this can be closed unless you want to leave it open for others for a bit.
